pkgs/node/src/chain.zig

```diff
@@ -328,10 +340,11 @@ pub const BeamChain = struct {
                 //check if we have the block already in forkchoice
                 const hasBlock = self.forkChoice.hasBlock(block_root);
 
-                self.module_logger.info("chain received gossip block for slot={any} blockroot={any} hasBlock={any}", .{
-                    //
+                self.module_logger.info("{}chain received gossip block for slot={any} blockroot={any} proposer={d} hasBlock={any}", .{
+                    zeam_utils.OptionalNode.init(self.getNodeNameForLog(block.proposer_index)),
```

Comment:
this is this node's log, node registry is not to print the name of this node, but to print the objects recieved from which node so one can see what our peers are sending us to easily identify who is sending what to us,

so this log should read: chain received gossip block from node={} for slot={}... and we fill the name of the node there from the node registery